tuned/plugins/plugin_disk.py

```diff
@@ -11,7 +11,78 @@
 
 class DiskPlugin(hotplug.Plugin):
 	"""
-	Plugin for tuning options of disks.
+	`disk`::
+	
+	Plug-in for tuning various block device options. This plug-in can also
+	dynamically change the advanced power management and spindown timeout
+	setting for a drive according to the current drive utilization. The
+	dynamic tuning is controlled by the Boolean [option]`dynamic` and
+	the global [option]`dynamic_tuning` option in `tuned-main.conf`.
+	+
```

Comment:
The dynamic tuning is controlled by the [option]`dynamic` and the global [option]`dynamic_tuning` option in `tuned-main.conf`.